config/environments/production.rb

```diff
@@ -74,22 +71,16 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
-  # Do not dump schema after migrations.
-  config.active_record.dump_schema_after_migration = false
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  if ENV['RAILS_HOST'].present?
-    config.action_mailer.default_url_options = { :host => ENV['RAILS_HOST'], :port => ENV['RAILS_PORT'] }
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
+    logger           = ActiveSupport::Logger.new(STDOUT)
+    logger.formatter = config.log_formatter
+    config.logger = ActiveSupport::TaggedLogging.new(logger)
   end
 
-  config.action_mailer.delivery_method = :smtp
-  # SMTP settings
-  config.action_mailer.smtp_settings = {
-      :address              => ENV['SMTP_HOST'],
-      :port                 => 587,
-      :user_name            => ENV['SMTP_USER'],
-      :password             => ENV['SMTP_PASSWORD'],
-      :authentication       => "plain",
-      :enable_starttls_auto => true
-  }
-
```

Comment:
Removing the mail settings from all configs, including production, doesn't look right to me.

Reply:
That was indeed not intended...